docs/pricing.mdx

```diff
@@ -144,7 +144,7 @@ When using LangSmith hosted at smith.langchain.com, data is stored in GCP region
 
 ### Is LangSmith SOC 2 compliant?
 
-We are in the process of getting our getting our SOC 2 certification. We will update the page when we’re certified.
+We are SOC 2 Type II, GDPR, and HIPAA compliant. You can request more information about our security policies and posture at trust.langchain.com. Please note we only enter into BAAs with customers on our Enterprise plan.
```

Comment:
Edited this because even though this section of docs is no longer maintained, it still gets returned in Google searches.